templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -196,16 +196,34 @@ else
 {% if parallel_test_workers == 0 %}
     if [[ "$GITHUB_WORKFLOW" == "{{ plugin_camel_short | default("Pulp") }} Nightly CI/CD" ]]; then
         pytest -v -r sx --color=yes --pyargs {{ plugin_snake }}.tests.functional
+
+    {%- for item in additional_repos if item.pytest_pyargs | default(false) %}
+        pytest -v -r sx --color=yes --pyargs {{ item.pytest_pyargs }}
+    {%- endfor %}
```

Comment:
Can we combine these in a single pytest invocation? Maybe add an option to stop trying after 20 failed tests.

Reply:
I think having them separate is kind of nice, that way if there is a failure you have big clear blocks showing you exactly where it is instead of one big run. The stopping after some time sounds nice but its not really the goal of this work.

Reply:
The thing is that we stop processing after the first failed block. If we can combine test suites in one pytest instantiation, you would see all the failed tests you need to fix to make the CI green again.
Also we loose half of the "--parallel" benefit.
And now i realize the parallel tests suffer from the same restriction. They cannot be called in the same run with the non parallel ones.
I'm on the fence here.